etna/models/statsforecast.py

```diff
@@ -0,0 +1,795 @@
+                **self.kwargs,
+            ),
+        )
+
+    def params_to_tune(self) -> Dict[str, BaseDistribution]:
+        """Get default grid for tuning hyperparameters.
+
+        This grid tunes parameters: ``order.0``, ``order.1``, ``order.2``.
+        If ``self.season_length`` is greater than one, then it also tunes parameters:
+        ``seasonal_order.0``, ``seasonal_order.1``, ``seasonal_order.2``.
+        Other parameters are expected to be set by the user.
+
+        Returns
+        -------
+        :
+            Grid to tune.
+        """
+        grid: Dict[str, BaseDistribution] = {
+            "order.0": IntDistribution(low=1, high=6),
+            "order.1": IntDistribution(low=1, high=2),
+            "order.2": IntDistribution(low=1, high=6),
+        }
+
+        num_periods = self.season_length
+        if num_periods > 1:
+            grid.update(
+                {
+                    "seasonal_order.0": IntDistribution(low=0, high=2),
+                    "seasonal_order.1": IntDistribution(low=0, high=1),
+                    "seasonal_order.2": IntDistribution(low=0, high=1),
+                }
+            )
+
+        return grid
+
+
+class StatsForecastAutoThetaModel(
+    PerSegmentModelMixin, PredictionIntervalContextIgnorantModelMixin, PredictionIntervalContextIgnorantAbstractModel
+):
+    """
+    Class for holding :py:class:`statsforecast.models.AutoTheta`.
+
+    Method ``forecast`` only works on ouf-of-sample data that goes right after training data.
+
+    Method ``predict`` only works on in-sample data.
+    """
+
+    def __init__(
+        self,
+        season_length: int = 1,
+        decomposition_type: str = "multiplicative",
+        model: Optional[str] = None,
+    ):
+        """Init model with given params.
+
+        Parameters
+        ----------
+        season_length:
+            Number of observations per unit of time. Ex: 24 Hourly data.
+        decomposition_type:
+            Sesonal decomposition type, 'multiplicative' (default) or 'additive'.
+        model:
+            Controlling Theta Model. By default searches the best model.
+        """
+        self.season_length = season_length
+        self.decomposition_type = decomposition_type
+        self.model = model
+        super().__init__(
+            base_model=_AutoThetaAdapter(
+                season_length=self.season_length, decomposition_type=self.decomposition_type, model=self.model
+            ),
+        )
+
+
+class StatsForecastAutoCESModel(
+    PerSegmentModelMixin,
+    NonPredictionIntervalContextIgnorantModelMixin,
+    NonPredictionIntervalContextIgnorantAbstractModel,
+):
+    """
+    Class for holding :py:class:`statsforecast.models.AutoCES`.
+
+    Method ``forecast`` only works on ouf-of-sample data that goes right after training data.
+
+    Method ``predict`` only works on in-sample data.
+    """
+
+    def __init__(self, season_length: int = 1, model: str = "Z"):
```

Comment:
Can we specify model via lateral for example?

Reply:
Here I used the type from `statsforecast` library. For this model the possible values are: `["Z", "N", "S", "P", "F"]` according to the [code](https://github.com/Nixtla/statsforecast/blob/main/statsforecast/ces.py#L631).

I'm not sure if we should fix that in our code.